### PR TITLE
Create hp.md

### PR DESCRIPTION
**Pull request by rachblevine**
Created new partner page for HP (which is different from HPE)

### CODE REVIEW

**Comment by rachblevine** [comment]
Thanks!

File: app/_partner/hp.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+facebook:
+hoursname:
+hourslink:
+
+
+flickr-apikey:
+flickr-setId:
+
+primary-hashtag: hp
+subhashtags: hpinspiresgiving
```
Comment (danbjoseph) [inline app/_partner/hp.md]: I think this may have to be a list, even if there is just one.

File: app/_partner/hp.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+layout: partner
+lang: en
+permalink: /hp/
+
+id: hp
+name: hp
+logo:
+contact:
+
+flickr: https://www.flickr.com/
```
Comment (danbjoseph) [inline app/_partner/hp.md]: I think this has to be the URL of an album and not just a link to the main flickr website.

File: app/_partner/hp.md (hunk 1 above)
Comment (rachblevine) [reply app/_partner/hp.md]: This is how it was done on the HPE site I copied & edited this from, but no clue.  Happy to take your advice!

File: app/_partner/hp.md (hunk 2 above)
Comment (rachblevine) [reply app/_partner/hp.md]: This button is just a place holder for them to see what they can customize, so we'll get a proper link if they choose to 1) use it and 2) provide an album.  Good catch!

File: app/_partner/hp.md (hunk 1 above)
Comment (dakotabenjamin) [reply app/_partner/hp.md]:
```
subhashtags:
  - hpinsiregiving
```

Is the correct format.

File: app/_partner/hp.md (hunk 2 above)
Comment (dakotabenjamin) [reply app/_partner/hp.md]: I recommend leaving the `flicker:` tag blank, but don't remove the whole line. I.e.
```
flickr:
```

**Pull request merged**